schemas/rum/view-schema.json

```diff
@@ -360,6 +360,40 @@
             }
           },
           "readOnly": true
+        },
+        "display": {
+          "type": "object",
+          "description": "Display properties",
+          "properties": {
+            "scroll": {
+              "type": "object",
+              "description": "Scroll properties",
+
+              "properties": {
+                "max_depth": {
+                  "type": "number",
+                  "description": "Maximum scrolled distance on this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_depth_scroll_height": {
+                  "type": "number",
+                  "description": "Page scroll height when the maximum scrolled distance was reached for this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_depth_scroll_top": {
+                  "type": "number",
+                  "description": "Page scroll top when the maximum scrolled distance was reached for this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_depth_time": {
+                  "type": "number",
+                  "description": "Timestamp of the scroll event that reached the maximum scroll depth for this view",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏for consistency with other descriptions
```suggestion
                "max_depth_time": {
                  "type": "number",
                  "description": "Timestamp of the scroll event that reached the maximum scroll distance for this view",
```